cocos2d/core/sprites/CCSpriteWebGLRenderCmd.js

```diff
@@ -24,20 +24,44 @@
 
 //Sprite's WebGL render command
 (function() {
+    var matrixByteSize =  4 * 4 * 4; //4 rows of 4 floats, 4 bytes each
```

Comment:
I can't use `matrixBytes` in `_forwardBatch` of RendererWebGL, because it's actually `matrixPerUnit in unsigned int 32`, so I will add some more comments